tensorboard/webapp/metrics/views/card_renderer/image_card_component.ng.html

```diff
@@ -21,6 +21,9 @@
       title="{{ tag }}"
       value="{{ title }}"
     ></tb-truncated-path>
+    <vis-selected-time-warning
```

Comment:
I'm a little confused: Isn't this still called vis-selected-time-clipped?

http://google3/third_party/tensorboard/webapp/metrics/views/card_renderer/vis_selected_time_warning_component.ts;l=21;rcl=435097075